doc/conda-forge.md

```diff
@@ -0,0 +1,113 @@
+to all `conda install` commands.
+
+### Create an environment and install dependencies
+Differently from `apt` and `homebrew`, the `conda` package manager is an `environment`-oriented package manager, meaning that packages are not 
+installed in some global location, but rather you install packages in an `environment` (that is just a directory in your filesystem), so that you
+can easily have multiple different environments with different packages installed on your system. To read more about this, check https://docs.conda.io/projects/conda/en/4.6.1/user-guide/tasks/manage-environments.html .
+
+For this reason, to compile the superbuild it is suggested to first create a conda environment, and then install in it all the dependencies
+required by the robotology-superbuild. To create a new environment called `robsub`, execute the following command:
+~~~
+conda create -n robsub
+~~~
+
+Once you created the `robsub` environment, you can "activate" it for the current terminal (i.e. make sure that the installed packages can be found) by the command:
+~~~
+conda activate robsub
+~~~
+
+**IMPORTANT: if you open a new terminal, you need to manually activate the environment also there. If you compiled a robotology-superbuild in a given conda environment, remember to activate it before trying to compile or run any package 
+of the robotology-superbuild.** 
+
+Once you activated it, you can install packages in it. In particular the dependencies for the robotology-superbuild can be installed as:
+~~~
+conda install cmake compilers make ninja pkg-config
+conda install ace asio boost eigen gazebo glew glfw gsl ipopt libjpeg-turbo libmatio libode libxml2 opencv pkg-config portaudio qt sdl sdl2 sqlite tinyxml
+~~~
+
+If you are on **Linux**, you also need to install also the following packages:
+~~~
+conda install expat-cos6-x86_64 freeglut libselinux-cos6-x86_64 libxau-cos6-x86_64 libxcb-cos6-x86_64 libxdamage-cos6-x86_64 libxext-cos6-x86_64 libxfixes-cos6-x86_64 libxxf86vm-cos6-x86_64 mesalib mesa-libgl-cos6-x86_64
+~~~
+
+If you are on **Windows**, you also need to install also the following packages:
+~~~
+conda install freeglut
+~~~
+
+### Clone the repo
+To compile the `robotology-superbuild` code itself, you need to clone it, following the instructions in https://github.com/robotology/robotology-superbuild#clone-the-repo . 
+
+### Compile the robotology-superbuild
+In a terminal in which you activate the `robsub` environment, you can compile.
+
+On **Linux** or **macOS**, run:
+~~~
+cd robotology-superbuild
+mkdir build
+cmake ..
+cmake --build . --config Release 
+~~~
+
+
+On **Windows**, run:
+~~~
+cd robotology-superbuild
+mkdir build
+cmake -G"Visual Studio 16 2019" ..
+cmake --build . --config Release 
+~~~
+
+**IMPORTANT: conda-forge does not provide Debug version of its libraries, so in Windows you can't compile in Debug mode if you are using conda-forge.**
```

Comment:
Maybe it is worth adding it to the disadvantages

Reply:
Good point.